R/dettl_runner.R

```diff
@@ -10,12 +10,17 @@
 #'
 dettl <- function(path, db_name = NULL) {
   mode <- get_mode(path)
-  if (!(mode %in% c("create", "append"))) {
+  if ((mode %in% c("create", "append"))) {
+    import <- RImport$new(path, db_name)
+  } else if (mode == "sql") {
+    import <- SqlImport$new(path, db_name)
```

Comment:
can the SQL imports be create/append types? I guess I will find out below, though it does feel like these are different "axes" of types...

Reply:
Yeah perhaps language and mode.
Really create/append is a choice about how it can effect the db and sql vs others is purely how the import will be written